payments-core/src/main/java/com/stripe/android/cards/CardAccountRangeService.kt

```diff
@@ -21,6 +23,7 @@ class CardAccountRangeService(
     val staticCardAccountRanges: StaticCardAccountRanges,
     private val accountRangeResultListener: AccountRangeResultListener,
     private val isCbcEligible: () -> Boolean,
+    private val cardBrandFilter: CardBrandFilter = DefaultCardBrandFilter
```

Comment:
Before we move this to private beta (not this PR), we will want and end to end/integration test ensuring this gets propagated all the way from the configuration.

Reply:
Agree!